Viewer/SPTPersistentCacheViewer.xcodeproj/project.pbxproj

```diff
@@ -239,112 +241,32 @@
 /* Begin XCBuildConfiguration section */
 		0510FF251BA2FF7A00ED0766 /* Debug */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = 6931DA7D255081C800AF4AF8 /* project.xcconfig */;
```

Comment:
updated the viewer to use the shared xcconfig - removed a lot of redundant config from the project